.github/workflows/ci.yml

```diff
@@ -9,6 +9,9 @@ on:
 env:
   DOCKER_TAG: buildenv
   DOCKER_FILE_PATH: ./docker-build-env/Dockerfile
+  TOKEN_LABEL: token-label
```

Comment:
In order to do proper testing, we would need to build the non-hsm version in a container that doesn't have the p11 libraries installed. Otherwise, you won't know if you code/build scripts silently assumes their existence in some cases.

Reply:
Done.